Resfeber/Model/Data.swift

```diff
@@ -0,0 +1,14 @@
+//
+//  Data.swift
+//  Resfeber
+//
+//  Created by Joshua Rutkowski on 11/7/20.
+//  Copyright © 2020 Spencer Curtis. All rights reserved.
+//
+
+import Foundation
+
+class Data {
```

Comment:
Can we pick a better name for this class? Data could mean anything

Reply:
🤦‍♂️ I knew this would be found...